install/kubernetes/cilium/values.yaml

```diff
@@ -2,12 +2,11 @@
 # This file is based on install/kubernetes/cilium/*values.yaml.tmpl.
 
 
-# upgradeCompatibility helps users upgrading to ensure that the configMap for
+# -- upgradeCompatibility helps users upgrading to ensure that the configMap for
```

Comment:
Why is `--` removed here?

Reply:
It was added, not removed. It causes `helm-docs` to add the comment for the option to the chart README.

Reply:
:facepalm: sorry for the noise